homework23/task_1.py

```diff
@@ -0,0 +1,31 @@
+class Person :
+    def __init__(self,name,age):
```

Comment:
space პარამეტრების შემდეგ, ისევ file formatter გამოიყენე და ამას ავტომატურად იზამს IDE
```suggestion
    def __init__(self, name, age):
```